### PR TITLE
[MRG] What is new? Everything!

### PR DESCRIPTION
**Pull request by betatim**
Started on a `whats_new.md`.

Thoughts?

I don't know into how much detail we want to go for 0.1. Basically everything is new :)

For future releases I think we should add a line in `whats_new.md` as part of the PR and then directly tag it with the name of the contributor(s). For this time I listed everyone who made one or more commits (sorted by surname).

### CODE REVIEW

**Comment by betatim** [comment]
If there are more things to list please post them here (or merge and then make your own PR)

**Comment by glouppe** [comment]
`whats_new.md` -> `HISTORY.md`?

**Comment by betatim** [comment]
¯\_(ツ)_/¯ your call, I like whats_new, HISTORY is so ... historical.

**Comment by glouppe** [comment]
`RELEASES.md`?

**Comment by glouppe** [comment]
or `CHANGELOG.md`? (this is the one I prefer)

File: whats_new.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+## Version 0.1
+
+First light!
+
+### New features
+
+* gaussian process based SMBO
+* ensemble of and single decision tree based SMBO
```
Comment (glouppe) [inline whats_new.md]: I would avoid acronyms

**Comment by betatim** [comment]
Done. Added one line to the `CONTRIBUTING.md` about pep8 (Thereby not following our own guidelines about mixing things together 🙈)

**Comment by glouppe** [comment]
Merging this, I'll make a very more changes directly.

**Pull request merged**